This is used for Sun Studio because the depend files are different. 
Can potentially remove this patch, but need to investigate to see when 
link_deps is run. 

--- mozilla-esr31/python/mozbuild/mozbuild/action/link_deps.py.orig	2015-06-04 17:42:52.548434964 -0700
+++ mozilla-esr31/python/mozbuild/mozbuild/action/link_deps.py	2015-06-04 17:42:52.553642029 -0700
@@ -129,9 +129,9 @@
             if deps:
                 deps = list(self.normpaths(deps))
                 for t in self.normpaths(rule.targets()):
-                    if t in self._targets:
-                        raise Exception('Found target %s in %s and %s'
-                                        % (t, self._targets[t][0], depfile))
+#                   if t in self._targets:
+#                       raise Exception('Found target %s in %s and %s'
+#                                       % (t, self._targets[t][0], depfile))
                     self._targets[t] = (depfile, deps)
 
     def dump(self, fh, removal_guard=True):
@@ -216,7 +216,8 @@
                               group=group,
                               abspaths=opts.abspaths)
     for f in opts.dependency_files:
-        linker.add_dependencies(open(f))
+        if os.path.isfile(f):
+            linker.add_dependencies(open(f))
 
     if opts.output:
         out = open(opts.output, 'w')
